lib/matplotlib/tests/test_figure.py

```diff
@@ -1082,3 +1083,31 @@ def test_add_axes_kwargs():
     assert ax1.name == 'rectilinear'
     assert ax1 is not ax
     plt.close()
+
+
+def test_ginput(recwarn):  # recwarn undoes warn filters at exit.
+    warnings.filterwarnings("ignore", "cannot show the figure")
+    fig, ax = plt.subplots()
+    Timer(.1, fig.canvas.button_press_event,
+          (*ax.transData.transform((.1, .2)), 1)).start()
```

Comment:
I realize it's a single line, but for parity, I would still define a `single_press` function to go here.

Reply:
sure